app/src/main/assets/fido2_privileged_community.json

```diff
@@ -75,6 +75,22 @@
           }
         ]
       }
+    },
+    {
+      "type": "android",
+      "info": {
+        "package_name": "net.quetta.browser",
+        "signatures": [
+          {
+            "build": "release",
+            "cert_fingerprint_sha256": "BE:FE:E7:31:12:6A:A5:6E:7E:FD:AE:AF:5E:F3:FA:EA:44:1C:19:CC:E0:CA:EC:42:6B:65:BB:F8:2C:59:46:80"
+          },
+          {
+            "build": "release",
+            "cert_fingerprint_sha256": "32:57:D5:99:A4:9D:2C:96:1A:47:1C:A9:84:3F:59:D3:41:A4:05:88:45:83:FC:08:7D:F4:23:7B:73:3B:BD:6D"
```

Comment:
Why are there two fingerprints here? I suppose one is from Google Play but what would be the other one?

Reply:
Honestly, it has been a few weeks since I've made this commit. I used "App Manager" to get the sha256 values. I tested it again without that other fingerprint and it seems to be working fine. It must have been removed since I do not see that sha256 fingerprint in "App Manager".

Reply:
In that case this would be a security concern, I'd take it off;

BE:FE:E7:31:12:6A:A5:6E:7E:FD:AE:AF:5E:F3:FA:EA:44:1C:19:CC:E0:CA:EC:42:6B:65:BB:F8:2C:59:46:80 I can confirm is from the Play Store, but the other one I cannot.

Reply:
> In that case this would be a security concern, I'd take it off;
> 
> BE:FE:E7:31:12:6A:A5:6E:7E:FD:AE:AF:5E:F3:FA:EA:44:1C:19:CC:E0:CA:EC:42:6B:65:BB:F8:2C:59:46:80 I can confirm is from the Play Store, but the other one I cannot.

Yes, I did take it off and removed it on my end. I'll be pushing the commit with the removal to my version/repo once I am home tonight. And if you're not aware, Quetta also updated the browser to support autofill, I tested the browser with the latest commits in the custom built apk. I can confirm that the autofill works with bitwarden now.